src/routes/search.ts

```diff
@@ -12,6 +12,7 @@ export default async function place(fastify: FastifyInstance) {
     const queryParams: any = req.query;
     const { op, place_id: placeId, type, prefix: dataPrefix } = queryParams;
     const level = parseInt(queryParams.level);
+    const bustcache = parseInt(queryParams.fresh) === 1;
```

Comment:
can you choose consistent lanugage here? you're referring to the same thing in three different ways: "bust cache", "clear", and "fresh". pick one and be consistent so the code is easier to follow